app/models/User.coffee

```diff
@@ -13,6 +13,8 @@ UserLib = {
   broadName: (user) ->
     return '(deleted)' if user.deleted
     name = _.filter([user.firstName, user.lastName]).join(' ')
+    if features?.china
+      name = user.firstName
```

Comment:
Hmm, why do we want to do this? What about users who did split their names into two parts in the previous system?

Reply:
For students, lastName is just a letter, it doesn't make sense in China.

Teachers might split their names, we can tell them to modify it as we have already remove firstName input field on setting and register page